corehq/apps/data_dictionary/util.py

```diff
@@ -264,6 +266,14 @@ def save_case_property(name, case_type, domain=None, data_type=None,
         return gettext('Unable to save valid values longer than {} characters').format(max_len)
 
 
+def delete_case_property(name, case_type, domain):
+    try:
+        prop = CaseProperty.objects.get(name=name, case_type__name=case_type, case_type__domain=domain)
+    except CaseProperty.DoesNotExist as e:
```

Comment:
Hmm, I'll vote for a manual/predefined error here rather, otherwise internal errors "leak" to the user. Also, predefined error messages tend to be more user friendly :smile:

Reply:
That's a good point! I've replaced the error with a pre-defined message ([c1a26bd](https://github.com/dimagi/commcare-hq/pull/33956/commits/c1a26bde53ba54c9a01db21ec00841ef14a6c308)).